src/Components/Login/LoginForm.tsx

```diff
@@ -60,6 +63,9 @@ const LoginForm = () => {
     e.preventDefault();
     if (!isValid) return;
 
+    const userData = { email: email };
```

Comment:
```suggestion
    const userData = { email };
```
key와 변수명이 동일하면 이렇게 바꿀 수 있죠~